extensions/amp-tiktok/0.1/amp-tiktok.js

```diff
@@ -121,6 +129,7 @@ export class AmpTiktok extends AMP.BaseElement {
         return;
       }
       this.videoId_ = blockquoteOrNull.dataset.videoId;
+      this.oEmbedRequestUrl_ = blockquoteOrNull.dataset.cite;
```

Comment:
`blockquoteOrNull` suggests we may want optional chaining?
```suggestion 
      this.oEmbedRequestUrl_ = blockquoteOrNull?.dataset?.cite;
```